tests/test-umockdev.c

```diff
@@ -1821,8 +1821,9 @@ assert_delta_t(const struct timeval * first, const struct timeval * second, int
 {
     int delta = (second->tv_sec - first->tv_sec) * 1000;
     delta += (second->tv_usec - first->tv_usec) / 1000;
-    g_assert_cmpint(delta, >=, ms - 50);
-    g_assert_cmpint(delta, <=, ms + 50);
+    /* specific timing can never be guaranteed if the testbed is not controlled */
+    if (delta < ms - 50 || delta > ms + 50)
+        g_printf("Warning: delta %dms out of expected %dms -/+ 50ms range.", delta, ms);
```

Comment:
Same here -- if the test doesn't fail, it's pointless to run it in the first place. Let's rather skip it on slow architectures, or apply a factor.